src/HealthCheck.Core/Results/CheckResult.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace HealthCheck.Core.Results
+{
+    public class CheckResult : ICheckResult
+    {
+        public string CheckerName { get; set; }
+        public string SectionName { get; set; }
```

Comment:
`SectionName` is fine with me as long as it's optional.

Reply:
it is optional if i got your point